packages/lesswrong/components/ea-forum/voting-portal/ElectionAllocateVote.tsx

```diff
@@ -209,18 +245,18 @@ const ElectionAllocateVote = ({
     }
   }, [displayedResults, sortedResults, updateSort]);
 
-  const { Loading } = Components;
+  const { Loading, ForumIcon } = Components;
   return (
     <div className={classNames(classes.root, className)}>
       <div className={classes.controls}>
         <button
-          className={classNames(classes.button, classes.sortButton, {
-            [classes.buttonDisabled]: !canUpdateSort,
+          className={classNames(classes.sort, {
+            [classes.hidden]: !canUpdateSort,
           })}
           disabled={!canUpdateSort}
           onClick={updateSort}
         >
-          Sort high to low
+          <ForumIcon icon="NarrowArrowDown" className={classes.sortIcon} /> Sort descending
```

Comment:
I remember trying this and thinking it looked worse that way (like too much text). I'll have another look

Reply:
It may look better with your icon + design but I don't want to put in the time to change that at this point. I'll just try and make it look nice with the small line of text below

Reply:
I did end up changing the icon but I kept it on the right because it felt more natural for it to be above the column it's sorting, and I kept "descending"

![Screenshot 2023-11-30 at 22 53 51](https://github.com/ForumMagnum/ForumMagnum/assets/77623106/aff38ae0-bf94-423c-b151-6557d8eb5b7d)
![Screenshot 2023-11-30 at 22 54 18](https://github.com/ForumMagnum/ForumMagnum/assets/77623106/e1e06367-1017-45e5-a1b4-c3d0cea493ac)